isso/wsgi.py

```diff
@@ -149,7 +149,7 @@ def add_cors_headers(status, headers, exc_info=None):
             return start_response(status, headers.to_list(), exc_info)
 
         if environ.get("REQUEST_METHOD") == "OPTIONS":
-            add_cors_headers("200 Ok", [("Content-Type", "text/plain")])
+            add_cors_headers("200 OK", [("Content-Type", "text/plain")])
```

Comment:
Huh? This is not the problem. 

Have you actually verified that this fixes the issue?

Reply:
Yes, I tested it on my server.

Reply:
And you were able to reproduce the issue in the other pull request? Because I don't see how this change addresses the traceback reported there.

Reply:
Traceback (most recent call last):
  File "/home/sim/work2/local/lib/python2.7/site-packages/Werkzeug-0.14.1-py2.7.egg/werkzeug/contrib/fixers.py", line 152, in __call__
    return self.app(environ, start_response)
  File "./isso/wsgi.py", line 121, in __call__
    return self.app(environ, start_response)
  File "./isso/wsgi.py", line 152, in __call__
    add_cors_headers("200 Ok", [("Content-Type", "text/plain")])
  File "./isso/wsgi.py", line 149, in add_cors_headers
    return start_response(status, headers.to_list(), exc_info)
TypeError: http status must be a string
[pid: 27190|app: 0|req: 4/5] 176.122.190.86 () {54 vars in 747 bytes} [Sun Jun  9 04:40:53 2019] OPTIONS /count => generated 0 bytes in 0 msecs (HTTP/2.0 500) 0 headers in 0 bytes (0 switches on core 0)
Traceback (most recent call last):
  File "/home/sim/work2/local/lib/python2.7/site-packages/Werkzeug-0.14.1-py2.7.egg/werkzeug/contrib/fixers.py", line 152, in __call__
    return self.app(environ, start_response)
  File "./isso/wsgi.py", line 121, in __call__
    return self.app(environ, start_response)
  File "./isso/wsgi.py", line 152, in __call__
    add_cors_headers("200 Ok", [("Content-Type", "text/plain")])
  File "./isso/wsgi.py", line 149, in add_cors_headers
    return start_response(status, headers.to_list(), exc_info)
TypeError: http status must be a string
[pid: 27190|app: 0|req: 5/6] 176.122.190.86 () {54 vars in 883 bytes} [Sun Jun  9 04:40:54 2019] OPTIONS /?uri=%2Fbrowser%2F2018%2F01%2F23%2Fa-mania-you-cant-miss%2F&nested_limit=5 => generated 0 bytes in 0 msecs (HTTP/2.0 500) 0 headers in 0 bytes (0 switches on core 0)

Reply:
Sorry, I recently changed to sub-uri and so I am now not affected by the problem. So I reproduced the problem by using isso with a subdomain. This solution works: https://github.com/posativ/isso/issues/297#issuecomment-452643024  

So I edited the PR again.......